data_steward/cdr_cleaner/manual_cleaning_rules/update_questiona_answers_not_mapped_to_omop.py

```diff
@@ -0,0 +1,167 @@
+# coding=utf-8
+"""
+Update survey Questions and Answers not mapped to OMOP concepts.
+
+There are several survey questions and answers that are not getting properly mapped from the RDR into the CDR.
+There are two sources for this error:
+    > Odysseus attempted to introduce some “short codes” that were not implemented in the RDR.
+    > There were some concepts that were invalidated and changed around.
+To address this, we implemented a fix that maps from the source_values present in the data to the “real”
+concept_ids in the OMOP vocabulary. This includes leveraging the supplemental old_map_short_codes.CSV where,
+Odyseus provided the short codes, as well as a brief review of other outstanding unmapped PPI codes.
+"""
+import logging
+
+import bq_utils
+import constants.cdr_cleaner.clean_cdr as cdr_consts
+import sandbox
+
+LOGGER = logging.getLogger(__name__)
+
+OLD_MAP_SHORT_CODES_TABLE = 'old_map_short_codes'
+
+OLD_MAP_SHORT_CODES_TABLE_FIELDS = [
```

Comment:
Should we define this table in the `resources/fields` section?  Maybe consider changing mode to required.  This will be easier to relax to nullable in the future if it is needed.  Please add a `description` string for each field.  This will make maintenance easier for everyone.